install-dev/upgrade/sql/1.7.8.0.sql

```diff
@@ -30,3 +30,9 @@ ALTER TABLE `PREFIX_hook` ADD `active` TINYINT(1) UNSIGNED DEFAULT 1 NOT NULL AF
 ALTER TABLE `PREFIX_orders` ADD COLUMN `note` TEXT AFTER `date_upd`;
 
 ALTER TABLE `PREFIX_currency` CHANGE `numeric_iso_code` `numeric_iso_code` varchar(3) NULL DEFAULT NULL;
+
+ALTER TABLE `PREFIX_product_attribute` ADD `date_add` DATETIME NOT NULL DEFAULT NOW() AFTER `available_date`;
```

Comment:
In this case, it shouldn't be `NOW` You must do it in three requests. 

```suggestion
ALTER TABLE `PREFIX_product_attribute` ADD `date_add` DATETIME NULL AFTER `available_date`;
UPDATE `PREFIX_product_attribute` AS pa
INNER JOIN `PREFIX_product` AS p ON pa.id_product = p.id_product 
SET pa.date_add = p.date_add; 
ALTER TABLE `PREFIX_product_attribute` ADD `date_add` DATETIME NOT NULL DEFAULT NOW() AFTER `available_date`;
```